main/local_modules/frontend/static/js/geocharts.js

```diff
@@ -0,0 +1,16 @@
+function geo_chart(companies, target) {
```

Comment:
Identifier 'geo_chart' is not in camel case.<br>'geo_chart' is defined but never used.